src/components/AddComment.jsx

```diff
@@ -20,15 +20,20 @@ const AddComment = ({ topic, article_id }) => {
       username: user.username,
       body: currText,
       article_id: article_id,
-    }).then(({ data }) => {
-      setCommentRes(data.comment);
-      setSubmitted(true);
-    });
+    })
+      .then(({ data }) => {
+        setCommentRes(data.comment);
+        setSubmitted(true);
+      })
+      .catch((err) => {
+        console.log(err);
```

Comment:
Again, rather than just console.logging the error, we should feedback to the user that there has been an error.